inventory

```diff
@@ -6,7 +6,6 @@ etcd
 [OSEv3:vars]
 openshift_deployment_type=origin
 containerized=true
-openshift_image_tag=v3.7.0
```

Comment:
Can you just change it to 3.9?
`openshift_image_tag=v3.9.0`

Reply:
done